modules/openideal_content/openideal_content.module

```diff
@@ -45,15 +45,42 @@ function openideal_content_form_views_exposed_form_alter(&$form, FormStateInterf
     || $form['#id'] == 'views-exposed-form-challenges-all-challenges-page'
     || $form['#id'] == 'views-exposed-form-ideas-all-ideas-page') {
     $form['search']['#placeholder'] = t('Search');
-
     if ($form['#id'] == 'views-exposed-form-frontpage-front-page') {
-      $form['add_idea_button'] = [
-        '#type' => 'link',
-        '#title' => t('Add idea'),
-        '#url' => Url::fromRoute('node.add', ['node_type' => 'idea']),
-        '#attributes' => ['class' => ['frontpage-add-idea-button']],
-        '#access' => \Drupal::currentUser()->isAuthenticated(),
+      $tour_button = [
+        '#type' => 'container',
+        '#attributes' => [
+          'class' => ['js-tour-start-button'],
+        ],
+        'button' => [
+          '#type' => 'html_tag',
+          '#tag' => 'button',
+          '#value' => t('Take a tour'),
+          '#attributes' => [
+            'aria-pressed' => 'false',
+            'type' => 'button',
+            'class' => ['tour-button', 'front-page-tour-button'],
+          ],
+          '#access' => \Drupal::currentUser()->hasPermission('access tour'),
+          '#attached' => [
+            'library' => [
+              'tour/tour',
+            ],
+          ],
+        ],
+      ];
+
+      $form['right_section'] = [
+        '#type' => 'container',
+        '#attributes' => ['class' => ['frontpage-right-section-wrapper']],
         '#weight' => 999,
+        'tour_button' => $tour_button,
+        'add_idea_button' => [
+          '#type' => 'link',
+          '#title' => t('Add idea'),
+          '#url' => Url::fromRoute('node.add', ['node_type' => 'idea']),
+          '#attributes' => ['class' => ['frontpage-add-idea-button']],
+          '#access' => \Drupal::currentUser()->isAuthenticated(),
```

Comment:
Do we have some relevant permission for this?

Reply:
Yep.